source/Cosmos.IL2CPU/ILOpCodes/OpNone.cs

```diff
@@ -684,6 +685,11 @@ public override void DoInterpretStackTypes()
             StackPushTypes[0] = xTypeShift;
             return;
           }
+          if (xTypeValue == typeof(UIntPtr) && (xTypeShift == typeof(int) || xTypeShift == typeof(uint)))
```

Comment:
Please include the case `xTypeValue == typeof(IntPtr)`

Reply:
There is already

i just copied that if and pasted down a line and changed that to UIntPtr

Reply:
the one you see deleted below, was a duplicate of the one above it